### PR TITLE
Matching changes required by Code Review

### PR DESCRIPTION
**Pull request by lucianosarno**
We did changes required:

- Change images for HTML&CSS (like, dislike, share and others)
- Added "H*" hidden tags where we would not have them
- Changes screenshot for README file 
- Removed link for "highlights.css"
- Joined .css files into only one stylesheet

### CODE REVIEW

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️
****
### Hi @lucianosarno and @Prabhakarzx! 👋 👋 
You have done a great job with your project so far! 👏  👏 You have also done a great job implementing most of the changes requested by the previous TSE! 👍 There are just a few issues that need to be resolved before you can move on ⬇️ 
****

- [x] The [HTML validator](https://validator.w3.org) is still showing an issue. I have attached a photo of the issue below 👇 
Screen Shot 2020-06-09 at 8 11 00 PM
_hint: I believe this issue stems from the fact that you don't have `<meta> tag` defined in the head of your HTML doc. please take a look at this [article](https://www.w3schools.com/html/html_head.asp) for more information._

- [x] As requested by the previous TSE, please don't use screenshots of the original page. A screenshot is being used in this section 👇 
Screen Shot 2020-06-09 at 8 26 47 PM

- You may use a screenshot for the avatar of the uploader but the name and amount of subscribers should not be a screenshot.

- [x] Please update the information and the live demo link in your README file. When I click the on the link it is showing an error 👇 
Screen Shot 2020-06-09 at 8 13 53 PM
****

- I noticed that this is your second review. In this case, you should use the same pull request as the one you used for your first code review, or if you closed the original pull request by accident, you should add a link referencing the original pull request in the subsequent pull request. In your next review, please submit this pull request. You may add a comment summarizing the changes you have made 😄

****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by Prabhakarzx** [comment]
Dear Brittany, Thank You for checking things out!
We've done everything needed from the previous review. And html validator
was not showing any errors or warnings. However, I found your review to
make it look even better. These are some quick fixes. We're doing it.
Thanks!

**Comment by Prabhakarzx** [comment]
Dear Brittany, Thank You for the improvement suggestions!
We've checked all the suggested issues and now its all fine.
Thanks!

**Comment by Prabhakarzx** [comment]
Hello there!
Greeting of the day, We've done the required changes. Kindly have a look:

- Change images for HTML&CSS (Channel info)
- Updated README file (Live Demo Link not working fixed and added issue link)
- Added metadata with viewport and ecoding information.

Thanks!

File: css/style.css
Comment (peterrobert) [inline css/style.css:118]: ### [OPTIONAL]
I have noticed you have used a lot of id selectors. I would encourage you to be using class selectors. Only use id selectors for very specific use in your web page

File: index.html
Comment (peterrobert) [inline index.html]: Kindly make this a button. Its a requirement in this project.
**hint** you can use the <button> tag
share

File: index.html
Comment (peterrobert) [inline index.html]: let's give this element a padding-left to make a space between the title and the views

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle:  :stop_sign: 
Great job so far :+1:  You have made the changes required by the previous TSE :+1:  However there some few issues that need to get fixed before we can approve this project . Kindly fix the suggested changes :arrow_down: 

- [ ] There are still an error in the HTML validator. Kindly fix this error :arrow_down: 
validations_html
**hint** try checking your  <h5> tag

I have added other required changes below.

**kindly fix them and resubmit** :pray: 

If you have any question about the review you can find me on slack using @peterrobert

### HAPPY CODING

File: index.html
Comment (Prabhakarzx) [reply index.html]: Thank You, Updated the button!

File: css/style.css
Comment (Prabhakarzx) [reply css/style.css:118]: Thank You! We'll keep in check!

File: index.html
Comment (Prabhakarzx) [reply index.html]: Thanks! Fixed the issue!

**Comment by Prabhakarzx** [comment]
Dear @peterrobert,
Thank You so much for your suggestions. Learned a lot so for.
We've fixed the suggested issues.
Have a great day ahead!
Thanks!

**Comment by Prabhakarzx** [comment]
### First of all Kindly accept our Thankfulness!
_We're very thankful that we've improved our coding skills and got some experience on handling the issues._
Thank You!

**Now, We've checked all the suggested issues.**

- Fixed HTML Validator Errors
- Made share as a clickable button
- Fixed some structural issues

Thanks!

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @lucianosarno and @Prabhakarzx, great job fixing the comments by the previous TSE's :+1: However, before the project can be approved, we need to fix the validator error.
- ``` <a href=""> ``` can not be a descendant of the ```<button> ``` it is the other way around
val

Kindly make this change and submit another code review :slightly_smiling_face: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Good job team. :fireworks: Keep it up.
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job :+1:
- You can now merge to master. I wish you all the very best in your next tasks :first_place_medal:
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Comment by Prabhakarzx** [comment]
### Hi @christianotieno and all the TSE team, mentors and fellow students,
**It was great experience working on the HTML Projects so for. We Learned so much in so many ways. We also tried inspect elements feature and developers tools.**
Kindly Accept our thankfulness.
We're very thankful for your kind reviews and appreciations.
Thanks You!

**Comment by lucianosarno** [comment]
I'm very thankful for my partner and for all MV team that stayed with us
along this project!
THanks a lot!!

Em qui, 11 de jun de 2020 2:13 AM, Prabhakar Pandey <
notifications@github.com> escreveu:

**Pull request merged**
